### ISSUE TITLE
RetroLink NES controller being incorrectly recognized

### ISSUE THREAD

**Issue by SarcasticJoe** [OpenEmu/OpenEmu#1188]
I just decided for fun to see if I could get my RetroLink NES (yes NES, not SNES) controller to work with OpenEmu some way, but low and behold it immediately recognized my controller. However the problem with this is that it incorrectly recognizes it as a RetroLink SNES controller.

The interesting part is that it does recognize up, down, left right, start select, A and B correctly, which at least in theory should let me play NES games with it. However the standard (and unchangeable) setting is to map is to map A to B and B to Y (which I do not have on my NES controller). In SNES games it on the other hand maps A to A and B to B. My assumption is that OpenEmu just assumes it's a RetroLink SNES controller because it sees a device with the manufacturer ID of "0x0079" and ignores the product ID of "0x0011" (which should at least be different than the product ID for the SNES version).

In other words it would be nice if you could remap buttons on the recognized controllers or if you could implement support for the RetroLink NES controller (which should be pretty easy with the information I posted).
